engine/runtime/src/main/scala/org/enso/interpreter/runtime/DefaultPackageRepository.scala

```diff
@@ -531,7 +547,10 @@ private class DefaultPackageRepository(
       key    <- keys
       module <- loadedModules.remove(key)
     } {
-      module.renameProject(newName)
+      module
+        .asInstanceOf[TruffleCompilerContext.Module]
+        .unsafeModule()
+        .renameProject(newName)
```

Comment:
same comment as above

Reply:
Maybe - [as in the other reply](https://github.com/enso-org/enso/pull/8144#discussion_r1371802616).